node_modules/col-core/lib/db.js

```diff
@@ -407,6 +407,126 @@ var setUpModel = function(sequelize) {
   Asset.belongsToMany(Category, {'through': 'assets_categories', 'onDelete': 'CASCADE'});
 
   /**
+   * The `whiteboards` table will keep track of the available whiteboards in the Whiteboard tool
+   *
+   * @property {String}       title                   The title of the whiteboard
+   */
+  var Whiteboard = module.exports.Whiteboard = sequelize.define('whiteboard', {
```

Comment:
This will be implemented as part of access management for whiteboards, which will be part of a follow-up PR.